sdk/nodejs/cmd/run/run.ts

```diff
@@ -150,39 +150,16 @@ export function run(
     }
 
     // If this is a typescript project, we'll want to load node-ts.
-    const typeScript: boolean = process.env["PULUMI_NODEJS_TYPESCRIPT"] === "true";
+    const typeScript = process.env["PULUMI_NODEJS_TYPESCRIPT"] === "true";
 
-    // We provide reasonable defaults for many ts options, meaning you don't need to have a tsconfig.json present
```

Comment:
I wonder if this needs to be saved for a major change version. There are potentially users who currently use pulumi with no tsconfig.json present and so get the default compiler options we currently set. With this change we are instead going to get ts-nodes default compiler options which I suppose is potentially breaking.

Reply:
Sure, I think https://github.com/pulumi/pulumi/pull/8199 is on the docket for release during the next major release as well, and the changes should build nicely off of each other.